app/assets/stylesheets/companion_window.css

```diff
@@ -4,6 +4,19 @@
   --color-fog-light: #f4f4f4;
 }
 
+.tooltip {
+  display: none;
+  color: #ddd;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 0.6rem;
+  background-color: #555;
+  padding: 0.3rem;
+  border-radius: 5px;
+  &.show {
+    display: block;
```

Comment:
`z-index: 1000;` or whatever you think makes more sense

Without this the tooltips get cut off in the header

![Screenshot 2025-02-13 at 5 40 13 PM](https://github.com/user-attachments/assets/8c23321e-a3d6-4c06-a403-53ba653f70eb)
![Screenshot 2025-02-13 at 5 39 56 PM](https://github.com/user-attachments/assets/1f43930c-0b6c-4d5d-98c4-f02157c62ea4)
![Screenshot 2025-02-13 at 5 39 53 PM](https://github.com/user-attachments/assets/b58f1055-8a3a-46a6-98fe-9bfc0004ec2d)

Reply:
I'm not seeing this cut off behavior. How are you getting it?

Reply:
Just by hovering with my mouse. Let me try in Chrome. I forgot my old version of Firefox does weird stuff.

Reply:
Yup. It is an old firefox issue. I am not seeing this behavior in chrome or safari